doc/tutorials/sandbox_config.ipynb

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">This notebook must be visited at least once in order to set up main configuration parameters.
</pre><p>This notebook must be visited first, before you start with the examples, in order to set up the main configuration parameters.</p><p> </p><pre class="ql-syntax" spellcheck="false">The configuration database is protected with a password. 
</pre><p>The configuration database is encrypted and protected with a password. </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>